apps/vscode/extension/package.json

```diff
@@ -51,11 +51,80 @@
 				]
 			}
 		],
+		"keybindings": [
+			{
+				"key": "cmd+numpad_add",
+				"title": "Zoom In",
+				"command": "tldraw.tldr.zoomIn",
+				"category": "tldraw",
+				"enablement": "resourceExtname == .tldr"
+			},
+			{
+				"key": "shift+cmd+=",
```

Comment:
This one doesn't work for me.

Reply:
We override this one but we don't actually use it in the app. So maybe we shouldn't override it.